test/contracts/airdrop/airdrop_utils.js

```diff
@@ -24,8 +24,9 @@ const Utils          = require('../../lib/utils.js'),
       Workers        = artifacts.require('./Workers.sol'),
       Airdrop        = artifacts.require('./Airdrop.sol'),
       EIP20TokenMock = artifacts.require('./EIP20TokenMock.sol'),
-      PriceOracle    = artifacts.require('./PriceOracleMock.sol')
-      ;
+      PriceOracle    = artifacts.require('./PriceOracleMock.sol'),
+      Web3 = require('web3'),
```

Comment:
This can be singleton method to create web3 object. 
This way openst-base integration will be smooth.